examples/mobile-viewer/viewer.js

```diff
@@ -16,7 +16,7 @@
 
 'use strict';
 
-if (!PDFJS.PDFViewer || !PDFJS.getDocument) {
+if (typeof PDFJS == 'undefined' || !PDFJS.PDFViewer || !PDFJS.getDocument) {
```

Comment:
Use `===` (strict equality) here like we do elsewhere.

Reply:
Done.  Anything else?